apps/engmanagement/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -0,0 +1,10 @@
+import SkillRecordings, {
+  SkillRecordingsOptions,
+} from '@skillrecordings/skill-api'
+import prisma from '../../../db'
+
+export const skillOptions: SkillRecordingsOptions = {
+  prismaClient: prisma,
```

Comment:
the prisma client is generated at the project level so we need to pass that in to the api